projects/fsastorefrontlib/src/cms-components/my-account/order/order-details/order-detail-items/order-detail-items.component.spec.ts

```diff
@@ -0,0 +1,246 @@
+import { Component, DebugElement, Input } from '@angular/core';
```

Comment:
You don't need all these test cases and these mocks. In our implementation, we are just initializing order with one method getOrderDetails (only that method should be mocked).

Reply:
It is enough just to check if the order details are fetched or not. Let's keep these tests simple and test only things which are actually called in our implementation.

Reply:
Fixed